pkg/utils/vdpa_provider.go

```diff
@@ -25,5 +28,18 @@ func GetVdpaProvider() VdpaProvider {
 }
 
 func (defaultVdpaProvider) GetVdpaDeviceByPci(pciAddr string) (vdpa.VdpaDevice, error) {
-	return vdpa.GetVdpaDeviceByPci(pciAddr)
+	// the govdpa library requires the pci address to include the "pci/" prefix
+	fullPciAddr := "pci/" + pciAddr
+	vdpaDevices, err := vdpa.GetVdpaDevicesByPciAddress(fullPciAddr)
+	if err != nil {
+		return nil, err
+	}
+	numVdpaDevices := len(vdpaDevices)
+	if numVdpaDevices == 0 {
+		return nil, fmt.Errorf("no vdpa device associated to pciAddress %v", pciAddr)
```

Comment:
nit: change %v to %s as this is a string

Reply:
done